libkbfs/key_bundle_cache.go

```diff
@@ -53,8 +53,8 @@ func (k *KeyBundleCacheStandard) GetTLFWriterKeyBundle(
 	tlf tlf.ID, bundleID TLFWriterKeyBundleID) (*TLFWriterKeyBundleV3, error) {
 	cacheKey := keyBundleCacheKey{tlf, bundleID.String(), true}
 	if entry, ok := k.lru.Get(cacheKey); ok {
-		if rkb, ok := entry.(*TLFWriterKeyBundleV3); ok {
-			return rkb, nil
+		if rkb, ok := entry.(TLFWriterKeyBundleV3); ok {
```

Comment:
Should the varname by `wkb` instead of `rkb`?  (Pre-existing, but might as well fix now.)